package.json

```diff
@@ -16,7 +15,26 @@
     "url": "https://github.com/eugene2candy/nodejs-mentoring-program/issues"
   },
   "homepage": "https://github.com/eugene2candy/nodejs-mentoring-program#readme",
+  "dependencies": {
+    "@hapi/joi": "^16.1.8",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "express": "~4.16.1",
+    "express-joi-validation": "^4.0.3",
+    "http-errors": "~1.6.3",
+    "jade": "~1.11.0",
```

Comment:
Jade was renamed to `Pug` due to copyrights and you should use that instead.
Last version published under the name jade was from 5 years ago.
BTW pug/jade is the coolest template engine ever :sunglasses:  cool choice

You can see the deprecation notice here: https://www.npmjs.com/package/jade

Reply:
I checked this and now I used pug instead